org.gnu.emacs.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id": "org.gnu.emacs",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "emacs",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--filesystem=host",
+        "--filesystem=/tmp"
```

Comment:
What's wrong with using private TMPDIR?

Reply:
I just noticed that sudo -e places the temporary copy of the file into /var/tmp so we might want also that.